### ISSUE TITLE
Laplace transform of Piecewise doesn't work even though SymPy can compute the integral

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#10354]
Reported on Gitter:

``` py
In [27]: u2=Piecewise((1.,t<1),(0.5,t>=1))

In [28]: print(laplace_transform(u2, t, s))
LaplaceTransform(Piecewise((1.00000000000000, t < 1), (0.500000000000000, t >= 1)), t, s)

In [29]: laplace_transform(u2, t, s).rewrite(Integral).doit()
Out[29]:
           -s
1.0   0.5⋅ℯ
─── - ───────
 s       s
```

**Comment by ronanpaixao** [comment]
Got a different, still unevaluated result:

    In[34]: u2=Piecewise((1.,t<1),(0.5,t>=1))
    
    In[35]: print(laplace_transform(u2, t, s))
    LaplaceTransform(Piecewise((1.00000000000000, t < 1), (0.500000000000000, t >= 1)), t, s)
    
    In[36]: laplace_transform(u2, t, s).rewrite(Integral).doit()
    Out[36]: Integral(Piecewise((1.0*exp(-s*t), t < 1), (0.5*exp(-s*t), t >= 1)), (t, 0, oo))

    In[37]: sympy.__version__
    Out[37]: '0.7.6.1'

**Comment by asmeurer** [comment]
I ran mine on master so that's probably why.

**Comment by nishnik** [comment]
@asmeurer You got the right results. And it is not coming the same when I tried to replicate it:
```
In [3]: u2=Piecewise((1.,t<1),(0.5,t>=1))

In [4]: laplace_transform(u2, t, s).rewrite(Integral)
Out[4]: Integral(Piecewise((1.0*exp(-s*t), t < 1), (0.5*exp(-s*t), t >= 1)), (t, 0, oo))

In [5]: sympy.__version__
Out[5]: '1.0.1.dev'
```
What needs to be done here ?

**Comment by asmeurer** [comment]
It isn't working for me anymore either. Something recent must have broken it.

**Comment by asmeurer** [comment]
I didn't try super hard, but I can't find a commit where it works. Not sure what happened. Is the answer I "got" above correct?

**Comment by nishnik** [comment]
Yeah! That's correct.
Then, let me try `git bisect` again and see if I can find which commit broke it.

**Comment by oscargus** [comment]
A potential difference/reason may be that the Piecewise is rewritten such that the last region is "otherwise". At least in the current master.

Hence, it is not obvious how the range is determined for the last segment of the integral (just guessing for the reason).

**Comment by hanspi42** [comment]
This is solved: #25693
